src/styles/_catalog.scss

```diff
@@ -14,9 +14,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+    transition-property: width;
```

Comment:
did you change width anywhere?

Reply:
Frankly speaking, I don't understand why the 'transition-property' was requested if all "animations' were required to be smooth. According to the task, all changes under  ':hover' should be smooth with 300ms duration. So therefore what 'transition-property' should I use? Usage of 'transition-property: all' looks senseless because 'transition-property: all' is the default meaning

Could you kindly explain what exact  'transition-property' (transition-property: color or transition-property: transform or transition-property: all) do you ask me to use and why, because this change requirement isn't clear for me from the main task. Thank you in advance)

Reply:
@Bohdan-Maksymiuk @vkryvytskyy Hello. Could you please give me some feedback?))))

Reply:
`all` is default and it works, but it consumes bigger amount of browser properties and it should affect the low performance machines. So better to indicate props  directly